extensions/sql-migration/src/wizard/skuRecommendation/skuRecommendationPage.ts

```diff
@@ -579,6 +577,26 @@ export class SKURecommendationPage extends MigrationWizardPage {
 		await this.refreshCardText(false);
 	}
 
+	public async refreshAzureRecommendation(): Promise<void> {
+		// TODO - Add cardLoading at start of refresh sku recommendation.
+		// await this.startCardLoading();
+		// this._skuLastRefreshTimeText.value = constants.LAST_REFRESHED_TIME();
```

Comment:
will this be added later or can it be removed

Reply:
Message notification will be added later. Same with cardLoading.